static/app/components/events/interfaces/spans/utils.tsx

```diff
@@ -544,7 +544,26 @@ export function getMeasurements(
   }
 
   const {startTimestamp} = event;
+  return _getMeasurements(event.measurements, startTimestamp, generateBounds);
+}
+
+export function getTraceMeasurements(
+  trace: TraceFullDetailed,
+  generateBounds: (bounds: SpanBoundsType) => SpanGeneratedBoundsType
+): Map<number, VerticalMark> {
+  if (!trace.measurements || !trace.start_timestamp) {
```

Comment:
Is there a better way of doing this instead of redoing everything so that this is `startTimestamp` to match the event type?

Reply:
Hmm, not the prettiest solution but maybe you can do something like this:
```
export function getMeasurements(
  event: EventTransaction | TraceFullDetailed,
  generateBounds: (bounds: SpanBoundsType) => SpanGeneratedBoundsType
): Map<number, VerticalMark> {
  const startTimestamp =
    (event as EventTransaction).startTimestamp ||
    (event as TraceFullDetailed).start_timestamp;
  if (!event.measurements || startTimestamp) {
    return new Map();
  }
  ```
Basically check for existence of either `startTimestamp` or `start_timestamp` since it looks like thats the only logical difference for this function.